svc/pkg/nomad/standalone/monitor/src/monitors/alloc_plan.rs

```diff
@@ -26,5 +27,17 @@ pub async fn handle(
 	})
 	.await?;
 
+	if util_job::is_nomad_job_run(job_id) {
```

Comment:
we could theoretically create a different job name that's specific to ds so we don't spam the ds system with unhandled signals.

Reply:
Fixed upstream